lib/daru/view/adapters/highcharts/display.rb

```diff
@@ -14,6 +16,9 @@ def self.init_script(
     # Note: Don't reorder the dependent_js elements. It must be loaded in
     # the same sequence. Otherwise some of the JS overlap and doesn't work.
     js =  ''
+    js << "\n<style type='text/css'>"
```

Comment:
It is not js. Define separate method for css.

Reply:
I have created another method `generate_init_code_css` for loading css code. I have changed the variable name accordingly.